src/mono/wasm/wasm.targets

```diff
@@ -35,15 +35,17 @@
                            OutputPath="$(WasmPInvokeTablePath)" />
   </Target>
 
-  <UsingTask TaskName="DownloadTimeZoneData" AssemblyFile="$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'CreateWasmBundle', 'Debug', '$(NetCoreAppCurrent)', 'publish', 'CreateWasmBundle.dll'))"/>
+  <UsingTask TaskName="CompileTimeZoneData" 
+    AssemblyFile="$(CreateCreateWasmBundlesAssemblyPath)"/>
   <Target Name="LoadTimeZones" DependsOnTargets="CheckEnv">
-    <DownloadTimeZoneData
-      InputDirectory="$(ArtifactsObjDir)wasm/data/input"
-      OutputDirectory="$(ArtifactsObjDir)wasm/data/output"
-      Version="2020a" />
+    <CompileTimeZoneData
+      TimeZones="@(_TimeZonesToBundle)" 
+      InputDirectory="$([MSBuild]::NormalizePath('$(CreateWasmBundleDir)', '..', 'data'))"
+      OutputDirectory="$(ArtifactsObjDir)wasm/data/output" />
   </Target>
 
-  <UsingTask TaskName="CreateWasmBundle" AssemblyFile="$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'CreateWasmBundle', 'Debug', '$(NetCoreAppCurrent)', 'publish', 'CreateWasmBundle.dll'))"/>
+  <UsingTask TaskName="CreateWasmBundle"
+    AssemblyFile="$(CreateCreateWasmBundlesAssemblyPath)"/>
```

Comment:
`CreateCreateWasmBundlesAssemblyPath`? One Create to much?

Reply:
It's because the task name is `CreateWasmBundle` I could change the name so it's not as confusing.